drdroid_debug_toolkit/core/protos/assets/asset.proto

```
@@ -67,6 +67,7 @@ message ConnectorModelTypeOptions {
     DatadogDashboardAssetOptions datadog_oauth_dashboard_model_options = 25;
     GithubRepositoryAssetOptions github_repository_model_options = 26;
     GithubMemberAssetOptions github_member_model_options = 27;
+    GithubProjectAssetOptions github_project_model_options = 122;
```

Buf (1.71.0)

[error] cannot find `GithubProjectAssetOptions` in this scope

(COMPILE)